src/libsync/discovery.cpp

```diff
@@ -346,6 +390,10 @@ void ProcessDirectoryJob::processFile(PathTuple path,
     item->_previousSize = dbEntry._fileSize;
     item->_previousModtime = dbEntry._modtime;
 
+    if (!localEntry.renameName.isEmpty()) {
```

Comment:
looks like that if is not really needed ?

Reply:
True;)